.gitlab-ci.yml

```diff
@@ -38,7 +38,11 @@ test-server_api:
       done
 
 test-simple:
-    script: stack --no-terminal test --ta "-p simple"
+    script:
+    - stack --no-terminal test --ta "-p simple"
+    - apt-get update
+    - apt-get install --assume-yes libgoogle-perftools-dev
+    - cd test/datalog_tests/simple_ddlog/ && LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/ cargo build --features='profile'
```

Comment:
Let's try to also do something about this so we don't compile it twice. @mbudiu-vmw recently added the `test/datalog_tests/run-test.sh` script that can be used instead of `stack test`.

Reply:
Hm, I don't think this script will work here. For one, I don't see how the claim that it honors `CARGOFLAGS`(as stated in its header) is true, as it overwrites that variable unconditionally. But even if it did, there is the problem that Cargo's `--features` argument cannot be supplied multiple times (in 1.34, it works in 1.38 on my system). See https://gitlab.com/ddlog/differential-datalog-d-e-s-o/-/jobs/336749170

We could introduce a dedicated variable just for passing in the features to use, but that seems ugly. I am tempted to say we can just live with it. What do you think?

Reply:
Agreed.